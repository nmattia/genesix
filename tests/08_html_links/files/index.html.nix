{ html }:
[ (html.tags.a { href = ./page.html.nix; } "Go to some other page?") ]
