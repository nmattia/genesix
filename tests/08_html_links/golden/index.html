<?xml encoding="UTF-8" version="1.0"?><a href="page.html">Go to some other page?</a><!-- this is a comment!! -->